.eslintrc.json

```diff
@@ -29,6 +29,7 @@
         "@typescript-eslint/dot-notation": "off",
         "@typescript-eslint/naming-convention": "off",
         "@typescript-eslint/no-throw-literal": "off",
+        "@typescript-eslint/no-shadow": "off",
```

Comment:
Right, I turned it off to avoid having the same variable (with same value), with different names in React components state (`_name` and `name`), but I can add a specific exception for it.

Reply:
Ok. I remember this helping me once from accidentally using/not using a variable from outer scope. Error resulting from this might have been hard to debug.